Review : Hindi translation

179priyasoni opened the pull request:
I m review on language  C and also for BASH

madiedgar commented:
Hey @179priyasoni! 👋 

Some exciting news - Based on feedback, we've updated the way we're reviewing translations. I incorporated your changes from this PR into the recent merge (I added you as a co-author). I know you have been participating in Hacktoberfest, so please let me know if you did not receive credit for #253 on your Hacktoberfest's profile. 

Also! 😄  **We still need more Python reviews (for keywords and builtin functions).** Would really appreciate it if you would review those translations so we can make Hindi one of the first official Legesher translations! 🎉  

There were a couple of words in Bash and C that did not receive a review. Feel free to make another PR to complete these (hopefully it is a lot easier this time!).

179priyasoni commented:
I didn't receive a credit for #253 on hacktoberfest

On Sun, Oct 17, 2021, 7:58 AM Madison (Pfaff) Edgar <
***@***.***> wrote:

> Hey @179priyasoni <https://github.com/179priyasoni>! 👋
>
> Some exciting news - Based on feedback, we've updated the way we're
> reviewing translations. I incorporated your changes from this PR into the
> recent merge (I added you as a co-author). I know you have been
> participating in Hacktoberfest, so please let me know if you did not
> receive credit for #253
> <https://github.com/legesher/legesher-translations/pull/253> on your
> Hacktoberfest's profile.
>
> Also! 😄 *We still need more Python reviews (for keywords and builtin
> functions).* Would really appreciate it if you would review those
> translations so we can make Hindi one of the first official Legesher
> translations! 🎉
>
> There were a couple of words in Bash and C that did not receive a review.
> Feel free to make another PR to complete these (hopefully it is a lot
> easier this time!).
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/legesher/legesher-translations/pull/246#issuecomment-945037829>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AUNFYKZCPQLAYG6BKUINJILUHIYEJANCNFSM5F2RYIQA>
> .
>

madiedgar commented:
I apologize @179priyasoni. I thought that if I put you as a co-author you would receive credit. I do have a solution! Pull the latest updates to the `hindi-translation` branch and add your translation reviews to a few places that might have been overlooked. I updated the format for the translations to have each contributor add their review in a new line following the word (see example below where you are the third translator, so you are the third newline after the word).

```yml
else: "और"
  # ✅ @sshiv5768
  # ✅ @bhaveshgoyal182
  # ❓ @179priyasoni "अन्यथा" is better
```
**PR Suggestion # 1**
There are some programming languages that you reviewed that have keywords without review. If you could find all the keywords that do not have an emoji in front of your username and enter your translation review on those lines, that would be lovely! For example, Bash's keywords "esac" and "fi", and C's "switch", "case" look like this and would need to have a review (even if its a 🤔  )

```yml
esac: "esac"
  # ✅ @sshiv5768
  # ✅ @bhaveshgoyal182
  #   @179priyasoni
```

**PR Suggestion # 2**
The initial issue was for **Python** keywords and builtin functions. This is Legesher's main priority, as we are working through a MVP with the Python programming language. If you could complete the review of Python keywords and builtin functions that would be the biggest priority and incredibly helpful. 

Thank you again for your contributions! It's been a pleasure working with you 😄
